db/migrate/20240416051242_drop_devise_tables.rb

```diff
@@ -0,0 +1,4 @@
+class DropDeviseTables < ActiveRecord::Migration[7.1]
```

Comment:
Why do we need this?

Reply:
used Devise and made the table in previous commits, 

To roll back everything Devise did, followed their docs and this migration happened

Reply:
deleted